src/config/dark-sites.config

```diff
@@ -413,6 +415,7 @@ jaxcore.app
 jbzd.com.pl
 jekyllrb.com
 jellymar.io
+jeremypetch.com
```

Comment:
This page is still work in progress, please remove it. WIP pages aren't allowed on the list.
```suggestion
```